deltachat-jsonrpc/src/api.rs

```diff
@@ -1426,6 +1427,16 @@ impl CommandApi {
         Ok(contact_id.map(|id| id.to_u32()))
     }
 
+    /// Parses a vCard file located at the given path. Returns contacts in their original order.
+    async fn parse_vcard(&self, path: String) -> Result<Vec<VcardContact>> {
```

Comment:
> now that deltachat_contact_tools::parse_vcard(vcard) returns a single contact

I don't think that's true?

Of course we could make it return a single contact, but to me the possible churn because we change the API in the future seems worse that the UIs having to always select the the first element.

Reply:
ah yes sorry it was `Result<Vec<VcardContact>>` the Result what was removed and for some reason I read it as that the Vec was removed from the returning type